### PR TITLE
Clean up some of the descriptions

### PR DESCRIPTION
**Pull request by ektravel**
### Description

This PR:
- Fixes table formatting
- Changes "depends on X" to "depends on the X"
- Removed redundant "note"

This PR has:

- [x] been self-reviewed.

### CODE REVIEW

File: docs/operations/metrics.md [hunk 1]
```diff
@@ -171,12 +171,12 @@ The mode is decided using the values
...
-|----------------------|-------------------|------|
-| `true`               | `false` | `APPEND`|
-| `true`               | `true  ` | Invalid combination, exception thrown. |
-| `false`              | `false` | `REPLACE_LEGACY` (this is the default for native batch ingestion). |
- | `false`              | `true` | `REPLACE`|
+|`isAppendToExisting`|`isDropExisting`|Mode|
+|--------------------|----------------|----|
+|`true`|`false`|`APPEND`|
+|`true`|`true `|Invalid combination, exception thrown.|
+|`false`|`false`|`REPLACE_LEGACY` (This is the default for native batch ingestion.)|
```
Comment (techdocsmith) [inline docs/operations/metrics.md:178]:
```suggestion
|`false`|`false`|`REPLACE_LEGACY`. The default for JSON-based batch ingestion. |
```

**Review by techdocsmith — APPROVED** [review]
Approved after nit.

File: docs/operations/metrics.md [hunk 2]
```diff
@@ -217,28 +217,28 @@ batch ingestion emit the following metrics. These metrics are deltas for each em
...
+|`ingest/merge/cpu`|CPU time in Nanoseconds spent on merging intermediate segments.|`dataSource`, `taskId`, `taskType`, `groupId`, `tags`|Depends on the configuration. Generally a few minutes at most.|
 |`ingest/handoff/count`|Number of handoffs that happened.|`dataSource`, `taskId`, `taskType`, `groupId`, `tags`|Varies. Generally greater than 0 once every segment granular period if cluster operating normally.|
-|`ingest/sink/count`|Number of sinks not handoffed.|`dataSource`, `taskId`, `taskType`, `groupId`, `tags`|1~3|
-|`ingest/events/messageGap`|Time gap in milliseconds between the latest ingested event timestamp and the current system timestamp of metrics emission. If the value is increasing but lag is low, Druid may not be receiving new data. This metric is reset as new tasks spawn up.|`dataSource`, `taskId`, `taskType`, `groupId`, `tags`|Greater than 0, depends on the time carried in event. |
+|`ingest/sink/count`|Number of sinks not handed off.|`dataSource`, `taskId`, `taskType`, `groupId`, `tags`|1~3|
```
Comment (kfaraz) [inline docs/operations/metrics.md:229]: Nice one 😂

File: docs/operations/metrics.md [hunk 3]
```diff
@@ -303,14 +303,14 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 |`segment/size`|Total size of used segments in a data source. Emitted only for data sources to which at least one used segment belongs.|`dataSource`|Varies|
 |`segment/count`|Number of used segments belonging to a data source. Emitted only for data sources to which at least one used segment belongs.|`dataSource`|< max|
 |`segment/overShadowed/count`|Number of segments marked as unused due to being overshadowed.| |Varies|
-|`segment/unavailable/count`|Number of segments (not including replicas) left to load until segments that should be loaded in the cluster are available for queries.|`dataSource`|0|
-|`segment/underReplicated/count`|Number of segments (including replicas) left to load until segments that should be loaded in the cluster are available for queries.|`tier`, `dataSource`|0|
+|`segment/unavailable/count`|Number of segments, not including replicas, left to load until segments that should be loaded in the cluster are available for queries.|`dataSource`|0|
```
Comment (kfaraz) [inline docs/operations/metrics.md:306]:
```suggestion
|`segment/unavailable/count`|Number of unique segments left to load until all used segments are available for queries.|`dataSource`|0|
```

File: docs/operations/metrics.md [hunk 4]
```diff
@@ -303,14 +303,14 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
...
+|`segment/underReplicated/count`|Number of segments, including replicas, left to load until segments that should be loaded in the cluster are available for queries.|`tier`, `dataSource`|0|
 |`tier/historical/count`|Number of available historical nodes in each tier.|`tier`|Varies|
 |`tier/replication/factor`|Configured maximum replication factor in each tier.|`tier`|Varies|
 |`tier/required/capacity`|Total capacity in bytes required in each tier.|`tier`|Varies|
 |`tier/total/capacity`|Total capacity in bytes available in each tier.|`tier`|Varies|
 |`compact/task/count`|Number of tasks issued in the auto compaction run.| |Varies|
-|`compactTask/maxSlot/count`|Max number of task slots that can be used for auto compaction tasks in the auto compaction run.| |Varies|
+|`compactTask/maxSlot/count`|Max number of task slots available for auto compaction tasks in the auto compaction run.| |Varies|
```
Comment (kfaraz) [inline docs/operations/metrics.md:313]:
```suggestion
|`compactTask/maxSlot/count`|Maximum number of task slots available for auto compaction tasks in the auto compaction run.| |Varies|
```

**Review by kfaraz — APPROVED** [review]

**Pull request merged**